v4/feature/receipt/src/main/java/exchange/dydx/trading/feature/receipt/components/isolatedmargin/DydxReceiptIsolatedPositionMarginUsageView.kt

```diff
@@ -37,7 +37,6 @@ object DydxReceiptIsolatedPositionMarginUsageView : DydxComponent {
         val formatter: DydxFormatter,
         val before: Double? = null,
         val after: Double? = null,
-        val reverseDirection: Boolean = false,
```

Comment:
Need to keep this.  The two receipt lines are also used on trade input.  If you remove this, it breaks the trade input use case

Reply:
but shouldn't we be treating direction the same way in both cases?